admin/workspace-management/gpu-acceleration.md

```diff
@@ -3,22 +3,22 @@ title: "GPU acceleration"
 description: Learn how to use GPUs with workspaces.
 ---
 
-GPUs can be used for performing tasks such as machine learning within
-workspaces.
+Graphical processing units (GPUs) are useful with compute-intensive workloads,
+such as those involved with data science/machine learning projects.
 
-By default, users cannot create workspaces with GPUs until a site manager
-configures and enables this feature.
+You can allocate GPUs to workspaces once a site manager configures and enables
+this feature. Enabling GPU acceleration requires that you've configured your
+Kubernetes cluster appropriately and
+[scheduled GPUs](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/).
```

Comment:
I believe this should be `schedules GPUs`.

Reply:
Think the past tense is correct, but removed the extra words that made this confusing